pkg/controller/migmigration/task.go

```diff
@@ -386,7 +409,39 @@ func (t *Task) Run() error {
 		if err = t.next(); err != nil {
 			return liberr.Wrap(err)
 		}
+	case CreateDirectImageMigration:
+		// Create the DirectImageMigration CR
+		err := t.createDirectImageMigration()
+		if err != nil {
+			return liberr.Wrap(err)
+		}
+		if err = t.next(); err != nil {
+			return liberr.Wrap(err)
+		}
+	case WaitForDirectImageMigrationToComplete:
+		// Get the DirectImageMigration CR
+		dim, err := t.getDirectImageMigration()
+		if err != nil {
+			return liberr.Wrap(err)
+		}
+		if dim == nil {
+			return errors.New("DirectImageMigration not found")
+		}
 
+		completed, reasons := dim.HasCompleted()
+		t.Log.Info("is migrations", "name", dim.Name, "completed", completed, "phase", dim.Status.Phase)
```

Comment:
Feels like this info should perhaps be rolled into `t.SetProgress()` instead of just logged on every reconcile

Reply:
Create follow-up github issue @sseago

Reply:
Github issue for DIM progress reporting: https://github.com/konveyor/mig-controller/issues/796
@djwhatle When we implement this, should this be instead of the log, or in addition to it?

Reply:
@sseago I think it makes sense to log when the status of the DIM changes, but I'm not sure logging the full list of progress every 3 seconds is necessary.

Reply:
@djwhatle OK, so when I do the follow-on progress reporting update, I'll change this to `t.Log.V(4).Info...` -- making it a debug-only log.